ivadomed/config/config_pose.json

```diff
@@ -0,0 +1,94 @@
+{
+    "command": "train",
+    "gpu_ids": [0],
+    "path_output": "/data/example/labeling_t0test1",
+    "model_name": "pose_model",
+    "debugging": false,
+    "loader_parameters": {
+        "path_data": ["/data/data-multi-subject"],
+        "target_suffix": ["_heatmap0"],
+        "extensions": [".nii.gz"],
+        "roi_params": {
+            "suffix": null,
+            "slice_filter_roi": null
+        },
+        "contrast_params": {
+            "training_validation": ["T1w"],
+            "testing": [ "T1w"],
+            "balance": {}
+        },
+        "slice_filter_params": {
+            "filter_empty_mask": true,
+            "filter_empty_input": true
+        },
+        "slice_axis": "sagittal",
+        "multichannel": false,
+	    "soft_gt": true
+    },
+    "split_dataset": {
+        "fname_split": null,
+        "random_seed": 8,
+        "split_method" : "participant_id",
+        "data_testing": {"data_type": null, "data_value":[]},
+        "balance": null,
+        "train_fraction": 0.6,
+        "test_fraction": 0.2
+    },
+    "training_parameters": {
+        "batch_size": 1,
+        "loss": {
+            "name": "JointsMSELoss"
+        },
+        "training_time": {
+            "num_epochs": 10,
+            "early_stopping_patience": 100,
+            "early_stopping_epsilon": 0.001
+        },
+        "scheduler": {
+            "initial_lr":0.0005,
+            "lr_scheduler": {
+                "name": "CosineAnnealingLR",
+                "base_lr": 1e-6,
+                "max_lr": 1e-4
+            }
+        },
+        "balance_samples": false,
+        "mixup_alpha": null,
+        "transfer_learning": null  
+    },
+    "default_model": {
+        "name": "HourglassNet"
+    },
+    "FiLMedUnet": {
+        "applied": false,
+        "metadata": "contrasts",
+        "film_layers": [0, 1, 0, 0, 0, 0, 0, 0, 0, 0]
+    },
+    "uncertainty": {
+        "epistemic": false,
+        "aleatoric": false,
+        "n_it": 0
+    },
+    "postprocessing": {
+        "remove_noise": {"thr": -1},
+        "binarize_prediction": {"thr": 0.5},
+        "uncertainty": {"thr": -1, "suffix": "_unc-vox.nii.gz"},
+        "remove_small": {"unit": "vox", "thr": 3}
+    },
+    "evaluation_parameters": {
+        "target_size": {"unit": "vox", "thr": [20, 100]},
+        "overlap": {"unit": "vox", "thr": 3}
+    },
+    "transformation": {
+      "Resample":
+        {
+            "wspace": 256,
+            "hspace": 256,
+            "dspace": 1,
+            "flag_pixel": true

```

Comment:
Documentation will be needed before merging this PR :-) https://ivadomed.org/en/latest/configuration_file.html#resample

Reply:
@rezazad68 did you address @charleygros 's comment?

Reply:
Not yet, his previous comment is to eliminate this part and use the original ivadomed `Resample`.